staging/src/k8s.io/kubectl/pkg/cmd/portforward/portforward.go

```diff
@@ -137,6 +137,7 @@ func (f *defaultPortForwarder) ForwardPorts(method string, url *url.URL, opts Po
 		return err
 	}
 	dialer := spdy.NewDialer(upgrader, &http.Client{Transport: transport}, method, url)
+	// dialer, err := portforward.NewWebSocketDialer(method, url, opts.Config, opts.Ports)
```

Comment:
can you enable it here so we run it in the cI?

Reply:
resolving as outdated